src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius = 5;
+
+    public Circle() {
+        this.setName("circle");
```

Comment:
hardcode. Better to accept values as a constructor parameters

Reply:
Okay, I got it